src/app/Sidebar/UserInfo.js

```diff
@@ -57,6 +59,13 @@ const UserInfo = ({ intl, authenticated, authenticatedUser, user, ...props }) =>
               }}
             />
           </div>
+          <div>
+            <i className="iconfont icon-flashlight text-icon" />
+            {intl.formatMessage({
+              id: 'voting_power',
+              defaultMessage: 'Voting Power',
+            })}: {currentVotingPower} %
```

Comment:
Can you use `FormattedNumber` from `react-intl` instead of this as well?
Like here: https://github.com/busyorg/busy/blob/493639ed919abc5a70f3b9a8384ff7dd7fde9ce5/src/components/Reactions/ReactionsList.js#L48-L51